gapic/templates/%namespace/%name_%version/%sub/services/%service/async_client.py.j2

```diff
@@ -24,6 +24,12 @@ from google.api_core import retry_async as retries
 from google.auth import credentials as ga_credentials   # type: ignore
 from google.oauth2 import service_account              # type: ignore
 
+try:
```

Comment:
Since the macro depends on this as a pre-condition, I suggest we make this a macro too. Plus, we then don't have to duplicate code in the sync version.

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/1999/commits/a5d6109fb82fb2770afd54267d023c30484eaa0d and https://github.com/googleapis/gapic-generator-python/pull/1999/commits/109edd462b49315450ca2d6672dc408069834bdc